src/appimagetool.c

```diff
@@ -200,6 +203,10 @@ int sfs_mksquashfs(char *source, char *destination, int offset) {
         args[i++] = "-mkfs-time";
         args[i++] = "0";
 
+        for (guint sqfs_opts_idx = 0; sqfs_opts_idx < sqfs_opts_len; sqfs_opts_idx++) {
```

Comment:
Sure, the compiler will optimize this anyway, but it'd be good style to use `++sqfs_opts_idx`.